lessons/Data-Science/choosing-your-dataset/langs/en-us/index.adoc

```diff
@@ -78,7 +78,7 @@ The Data Cycle is a roadmap that guides us in the process of data analysis. You'
 *Analyze Data*
 @Q{We'll choose what kind of display we'll need to answer our question.}
 @Q{Which two visualizations work with categorical data? Why might you choose one over the other?}
-@A{@ifproglang{pyret}{Bar and pie charts work}@ifproglang{codap}{Dot plots and bar charts} with categorical data. @ifproglang{pyret}{A pie chart only makes sense when you have the full picture, whereas a bar chart shows the count.} @ifproglang{codap}{A dot plot conveys more precise information than a bar chart, but can be overwhelming if there is a lot of data}.}
+@A{@ifproglang{pyret}{Bar and pie charts work}@ifproglang{codap}{Dot plots and bar charts} with categorical data. @ifproglang{codap}{A dot plot conveys more precise information than a bar chart, but can be overwhelming if there is a lot of data}.}
```

Comment:
Move the word "work" out of @ifproglang{pyret} so that the sentence reads correctly for CODAP as well.
Add a pyret explanation for why you would choose a bar chart vs. a pie chart. I still think it's true that pie charts only make sense when the data we have represents the whole.

Reply:
done